extension/app/src/css/custom.css

```diff
@@ -1,4 +1,4 @@
 body {
   font-family: darkmode-off-cc, sans-serif;
-  font-size: 100%;
+  font-size: 14px !important;
```

Comment:
I don't think you need !important ?

Reply:
Looks like I need it in the end, in the convo it's still 16px 😬